ujin/programmers/day02/두-정수_사이의-합.js

```diff
@@ -0,0 +1,5 @@
+function solution(a, b) {
+  let abSum = 0;
+  abSum = ((a + b) * (Math.abs(b - a) + 1)) / 2;
+  return abSum;
```

Comment:
하나에 변수에 값을 할당하고 그 변수를 바로 return 하는 경우 값 자체를 return 해도 무방합니다.
```suggestion
  return ((a + b) * (Math.abs(b - a) + 1)) / 2;
```